ui/src/components/DenylistForm/index.jsx

```diff
@@ -169,7 +169,7 @@ export default class DenylistForm extends Component {
                 value={notificationType}
                 onChange={this.handleInputChange}
                 name="notificationType">
-                {notificationTypes.map(type => (
+                {Object.keys(DENYLIST_NOTIFICATION_TYPES).map(type => (
```

Comment:
Did you mean to do `Object.values` instead?

Reply:
They are the same in out case but yes `Object.values` is more appropriate